Resources/views/js.html.twig

```diff
@@ -23,6 +23,13 @@
             this.language = '{{ language }}';
         {%- endif -%}
 
+        {%- if callback is defined -%}
+            var fn = window['{{callback}}'];
+            if(typeof fn === 'function') {
+                this.callbacks.onNewComment = [function () { fn() }];
```

Comment:
Could we just do this? It would also mean that the `comment` argument is passed to their function (near the bottom of this): https://help.disqus.com/customer/portal/articles/466258-capturing-disqus-commenting-activity-via-callbacks

```js
this.callbacks.onNewComment = [fn]
```